README.md

```diff
@@ -17,3 +17,17 @@ Wouldn't it be nice to replace bash with a Clojure shell?
 Copyright © 2013-2016 Clojure-Aided Enrichment Center
 
 Apache License, Version 2.0.
+
+<!-- Named page links below: /-->
+
+[travis]: https://travis-ci.org/clojusc/shankha
+[travis-badge]: https://travis-ci.org/clojusc/shankha.png?branch=master
+[deps]: http://jarkeeper.com/clojusc/shankha
+[deps-badge]: http://jarkeeper.com/clojusc/shankha/status.svg
+[logo]: resources/images/clj-shankha-logo-3.png
```

Comment:
This line and the next one aren't actually correct, since the images in this repo are different from the repo where these badge links were copied from. See the comment above (at the top of the README) for more info.